src/animators/animator.ts

```diff
@@ -11,7 +46,8 @@ export interface Animator {
    *     transition object so that plots may chain the transitions between
    *     animators.
    */
-  animate(selection: d3.Selection<any>, attrToAppliedProjector: AttributeToAppliedProjector): d3.Selection<any> | d3.Transition<any>;
+  animate(selection: d3.Selection<any>|d3.Transition<any>, attrToAppliedProjector: AttributeToAppliedProjector,
+    drawingTarget?: Drawers.DrawingTarget, drawer?: Drawer): d3.Selection<any> | d3.Transition<any>;
```

Comment:
This is specifically the part that I think I will have questions with, mainly because it seems now `Drawer`s are leaking into the `Animator` and because `animate` is being much more complicated.